lib/Admin.php

```diff
@@ -625,7 +632,7 @@ public function fix_widen_attachment_url( $url, $attachment_id ) {
 		if ( ! empty( $widen_media_id ) ) {
 			$base_url             = Widen::$base_url;
 			$url_path             = self::get_widen_url_path( $url );
-			$widen_attachment_url = trailingslashit( $base_url ) . $url_path;
+			$widen_attachment_url = trailingslashit( $base_url ) . ltrim( $url_path, '/' );
```

Comment:
Using ltrim here to remove any potential leading slashes that may exist from previously saved images.